lib/pf/radius.pm

```diff
@@ -225,6 +226,12 @@ sub authorize {
     $args->{'ssid'} = $ssid;
     $args->{'node_info'} = $node_obj;
     $args->{'fingerbank_info'} = pf::node::fingerbank_info($mac, $node_obj);
+    my $filter = pf::access_filter::radius->new;
+    my $rule = $filter->test('fixup', $args);
```

Comment:
What is fixup ? It isn't a very explicit name about what this does

Perhaps pre_process or something like this